scripts/update-integration-test-coordinates.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+python integration-test.py -printcoords | python scripts/update-integration-test-coordinates.py
```

Comment:
It will be based on how it's run from circleci. The root of the repo is the assumed cwd.

What do you suggest instead? In the setup and run script I see this:

    basedir="$(dirname ${BASH_SOURCE[0]})/.."

Should we do the same thing here?